src/breeze/components/breeze/sidebar_links.cr

```diff
@@ -1,19 +1,32 @@
 class Breeze::SidebarLinks < BreezeComponent
   needs context : HTTP::Server::Context
 
-  # NOTE: https://github.com/luckyframework/lucky/issues/1243
-  # Habitat.create do
-  #   setting links : Hash(String, BreezeAction.class) = {
-  #     "Requests" => Breeze::Requests::Index,
-  #     "Queries"  => Breeze::Queries::Index,
-  #   }
-  # end
+  Habitat.create do
+    setting links : Array(Proc(Breeze::SidebarLinks, Nil)) = [
+      ->(breeze : Breeze::SidebarLinks) {
+        breeze.mount(Breeze::SidebarLink, context: breeze.context, link_text: "Requests", link_to: Breeze::Requests::Index)
```

Comment:
Without passing in the Component context, `mount` doesn't exist... neither does `context`.